tracer/build/_build/Build.Steps.cs

```diff
@@ -269,6 +271,23 @@ partial class Build
         .DependsOn(CompileNativeSrcMacOs)
         .DependsOn(CompileNativeSrcLinux);
 
+    Target CppCheckNativeSrcLinux => _ => _
+        .Unlisted()
+        .OnlyWhenStatic(() => IsLinux)
```

Comment:
Assume there's no equivalent for macos?

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/3483/commits/61ab074d31ea16f2f559ee9f790ac785a79a7144